### DISCUSSION TITLE
Werte per mqtt in iobroker einbinden

### DISCUSSION THREAD

**Discussion by Christian-4711** [tbnobody/OpenDTU#371] [General]
Hallo in die Runde!

Ich habe eine Frage an die iobroker Nutzer, weil ich offensichtlich zu blöd bin oder einen Denkfehler habe :-)

Die Werte der OpenDTU sollen per mqtt im iobroker landen.
Im iobroker habe ich den sonoff Adapter laufen und bekomme dort auch schon Werte von einem "Volkszähler" (Werte des Stromzählers über die optische Schnittstelle des Zählers).
Ich schaffe es gerade nicht, die Werte der OpenDTU zu empfangen. In der OpenDTU ist mqtt konfiguriert (Base Topic "solar/") und scheinbar klappt das auch irgendwie, denn im iobroker sehe ich ein "Alive = true":
![grafik](https://user-images.githubusercontent.com/111051502/203315183-667f874e-72f3-45ea-a5d0-198b7cd472c0.png)

Andere Werte sehe ich nicht. 
Woanders habe ich gelesen, dass ich die Topics manuell anlegen muss....habe ich auch mal teilweise gemacht, ändert aber nichts.
Ist dieser Baum dann so richtig?
![grafik](https://user-images.githubusercontent.com/111051502/203315726-5d9c4585-c3c1-401c-a8e7-de656f7a69d8.png)

Hab ich sonst irgendwas nicht bedacht? Habt ihr andere Tipps?

Danke im Voraus und viele Grüße,
Christian

**Comment by elwetritsche134** [comment]
Das Problem hatte ich damals auch. Installiere dir den Adapter "MQTT Broker/Client" und nutze einen anderen Port (z.B. 1884) als bei dem Sonoff Adapter. Dann sollte OpenDTU auch die Daten senden. Topics werden automatisch angelegt.

**Reply by Christian-4711** [reply]
Das funktioniert auf Anhieb. 
Ich danke Dir!
